PhysicsTools/PatAlgos/python/slimming/miniAOD_tools.py

```diff
@@ -512,8 +512,10 @@ def miniAOD_customizeData(process):
     from PhysicsTools.PatAlgos.tools.coreTools import runOnData
     runOnData( process, outputModules = [] )
     process.load("RecoCTPPS.TotemRPLocal.ctppsLocalTrackLiteProducer_cff")
+    process.load("RecoCTPPS.ProtonReconstruction.ctppsProtons_cff")
     task = getPatAlgosToolsTask(process)
     task.add(process.ctppsLocalTrackLiteProducer)
+    task.add(process.ctppsProtons)
```

Comment:
it looks (from the errors in tests of #26335 https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-26335/33931/runTheMatrix-results/4.53_RunPhoton2012B+RunPhoton2012B+HLTD+RECODR1reHLT+HARVESTDR1reHLT/step3_RunPhoton2012B+RunPhoton2012B+HLTD+RECODR1reHLT+HARVESTDR1reHLT.log)  like both pps additions better be added with ctpps_2016
```python
 ctpps_2016.toModify(task, func=lambda t: t.add(process.ctppsLocalTrackLiteProducer))
 ctpps_2016.toModify(task, func=lambda t: t.add(process.ctppsProtons))
```